R/Rules-class.R

```diff
@@ -139,13 +169,101 @@ validObject(.NextBestNCRM())
 ##' @keywords methods
 NextBestNCRM <- function(target,
                          overdose,
-                         maxOverdoseProb)
-{
-    .NextBestNCRM(target=target,
-                  overdose=overdose,
-                  maxOverdoseProb=maxOverdoseProb)
+                         maxOverdoseProb) {
+  .NextBestNCRM(
+    target = target,
+    overdose = overdose,
+    maxOverdoseProb = maxOverdoseProb
+  )
```

Comment:
I assume all the changes above and below are  just styler based changes.
Tip for next PR: better is to move the `#nolint end` before your additional class, add another `#nolint start` after your class. such that you only need to format your own class part, then the rest will be cleaned up later. that also simplifies the review

Reply:
In the updated code, I follow this instruction